docs/development/saml.md

```diff
@@ -1,10 +1,9 @@
 # How to test SAML as a developer without a server
 
-https://capriza.github.io/samling/samling.html (https://github.com/capriza/samling) is a great
-resource for being able to tinker with the SAML options within Synapse without needing to
-deploy and configure a complicated software stack.
+https://github.com/DFE-Digital/samling is a great resource for being able to tinker with the 
```

Comment:
This is unfortunately not really the same link. The old link pointed to a hosted version of it which was very useful. 😢 I don't know of anywhere that it is hosted online now though! I don't suppose that project has a GitHub pages version?

Reply:
You are completely right. Unfortunately, I was not able to find a hosted version out there :(. (I have removed the hosted link and changed the old GitHub reference to the "new" repo in my PR. This way, interested developers are at least able to fork the repo and start an own GitHub pages site for testing.)

Reply:
Somebody's done just that. There's a hosted version at https://fujifish.github.io/samling/samling.html.

Reply:
Thank you, I've included the GitHub pages URL.